files/galaxy/config/user_preferences_extra_conf.yml

```diff
@@ -64,6 +64,7 @@ preferences:
           # - ["Bergen (Norway) - UIB", no-pulsar]
           - ["Prague (Czech Republic) - MetaCentrum", cz-pulsar]
           - ["Bratislava (Slovakia) - IISAS", sk-pulsar]
+          - ["EGI Foundation", egi-pulsar]
```

Comment:
@enolfc What is the physical location of the server and the institution?

Reply:
The physical location will be Bari (Italy), supported by INFN.
Be aware that this cannot be considered a stable endpoint as we are now mostly trying to get the automated deployment working and we may move to other providers in the future. Let me know if we should go for another approach

Reply:
That's totally ok, we can change the text later.